smcontroller/smcontroller.go

```diff
@@ -48,15 +51,17 @@ const connectClientTimeout = 1 * time.Minute
 type Controller struct {
 	sync.Mutex
 
-	messageSender    MessageSender
-	alertSender      AlertSender
-	monitoringSender MonitoringSender
-	urlTranslator    URLTranslator
-	clientsWG        sync.WaitGroup
-	readyWG          sync.WaitGroup
-	clients          map[string]*smClient
-	context          context.Context // nolint:containedctx
-	cancelFunction   context.CancelFunc
+	messageSender            MessageSender
+	alertSender              AlertSender
+	monitoringSender         MonitoringSender
+	urlTranslator            URLTranslator
+	clientsWG                sync.WaitGroup
+	readyWG                  sync.WaitGroup
+	updateInstanceStatusChan chan []cloudprotocol.InstanceStatus
+	runStatusChan            chan unitstatushandler.RunInstancesStatus
+	clients                  map[string]*smClient
+	context                  context.Context // nolint:containedctx
```

Comment:
I guess we can remove context instead of suppressing lint. It can be passed as parameter.

Reply:
to fix lint i have renamed context to ctx